Remove top warnings and bump version to 1.0.0

nathandyer opened the pull request:
This PR removes the top warnings (references to the SDW Pilot) and also bumps the version number to 1.0.0.

Closes #209

legoktm commented on docs/general/known_issues.rst:
```diff
@@ -1,16 +1,14 @@
+via the `support portal <https://support.freedom.press>`_. 
+
+Bugs and other issues that are not specific to your instance can be reported
+via Github using the following links:
```
nit:

```suggestion
via GitHub using the following links:
```

Maybe also "If you encounter a security-related issue, please see [SECURITY.md](https://github.com/freedomofpress/securedrop-workstation/blob/main/SECURITY.md) for instructions on how to privately report it"??

nathandyer replied:
Great suggestion! Added :)

legoktm reviewed (changes requested):
Nice, only one thing re security issues.

legoktm commented on docs/general/status.rst:
```diff
@@ -1,18 +1,15 @@
-The general availability release is tentatively planned for Q3/Q4 2024.
+SecureDrop Workstation is currently in active development.
 
 Freedom of the Press Foundation has operated a pilot program with a limited
```
```suggestion
Freedom of the Press Foundation had operated a pilot program with a limited
```

legoktm commented on docs/general/status.rst:
```diff
@@ -1,18 +1,15 @@
-pilot participants will be considered or accepted, and current pilot
-participants can expect to transition to the standard support experience
-before the general availability release is made available later in 2024.
+early testing of the Workstation. This pilot program is now closed.
```
Maybe?

```suggestion
early testing of the Workstation. This pilot program has now ended.
```

nathandyer commented:
Thanks @legoktm! I believe I addressed your feedback. One note, I've never done the in-line commits before while also trying to rebase to make the commits look pretty, while still preserving the `co-authored by` information; all that is to say, if it's a mess or needs to be changed, let me know what needs to be done and I'll polish it up.

legoktm reviewed (approved):
:shipit:
